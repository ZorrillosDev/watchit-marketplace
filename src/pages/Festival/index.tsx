// react imports
import React from 'react';

// ===========================|| FESTIVAL ||=========================== //

const Festival = (): JSX.Element => {
  return <span>How it work</span>;
};

export default Festival;
