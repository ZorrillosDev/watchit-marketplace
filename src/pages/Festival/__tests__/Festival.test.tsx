import React from 'react'
import { shallow } from 'enzyme'
import Festival from '../index'

/* eslint-disable no-undef */
describe('<Festival />', () => {
  it('should render', () => {
    const component = shallow(<Festival />)
    expect(component).toMatchSnapshot()
  })
})
