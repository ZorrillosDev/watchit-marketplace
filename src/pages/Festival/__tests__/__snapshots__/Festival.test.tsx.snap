// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Festival /> should render 1`] = `ShallowWrapper {}`;
